netbox/core/views.py

```diff
@@ -232,3 +237,61 @@ def post(self, request, pk):
         messages.success(request, f"Restored configuration revision #{pk}")
 
         return redirect(candidate_config.get_absolute_url())
+
+#
+# Background Tasks (RQ)
+#
+
+
+class BackgroundQueuesListView(LoginRequiredMixin, View):
```

Comment:
List views should use the singular form of the object name for consistency.
```suggestion
class BackgroundQueueListView(LoginRequiredMixin, View):
```